### ISSUE TITLE
"Undefined" in violation message for at-rules

### ISSUE THREAD

**Issue by jeddy3** [csstools/stylelint-prefer-nesting#2]
Thanks for a great plugin! It's already caught inconsistencies in our code base since we started using it last week.

Since, `1.2.0` the plugin warns about unnested at-rules. However, the violation message isn't as helpful as the one that accompanies unnested rules.

```css
.class {}

@media (min-width: 48rem) {
  .class {}
}
```

Produces:

```console
1:1   ✖  Expected ".class" inside "undefined".   csstools/use-nesting
```

Whereas:

```css
.class {}

.class:hover {}
```

Produces:

```console
3:1   ✖  Expected ".class:hover" inside ".class".   csstools/use-nesting
```

Is it possible to improve the at-rule violation message so that the at-rule is mentioned by name e.g. `Expected "@media" inside ".class"`?

**Comment by jonathantneal** [comment]
Yes it is, @jeddy3. Thanks for bringing this to my attention. I can also improve my tests to cover the usability of its warning messages.

**Comment by jonathantneal** [comment]
Resolved by https://github.com/csstools/stylelint-use-nesting/commit/38588633b18307ec1bae3f9c0323830405c6739a

Published in v1.2.1! 🎉

**Comment by jeddy3** [comment]
> Published in v1.2.1! 🎉

Fantastic, thank you very much!
